test/hca/components/HealthCareApp.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I think this was brought in in https://github.com/department-of-veterans-affairs/vets-website/pull/2703/files#diff-a8af7004dab28f2e0d50acf7e7f31b6dR1, just renamed. Maybe don't need this then?